apps/condo/domains/notification/schema/DisconnectUserFromDeviceService.js

```diff
@@ -0,0 +1,51 @@
+/**
+ * Generated by `createservice notification.DisconnectUserFromDeviceService --type mutations`
+ */
+const get = require('lodash/get')
+
+const { GQLCustomSchema } = require('@core/keystone/schema')
+
+const access = require('@condo/domains/notification/access/DisconnectUserFromDeviceService')
+const { Device: DeviceAPI } = require('@condo/domains/notification/utils/serverSchema')
+
+const { PUSH_TRANSPORT_TYPES } = require('../constants/constants')
+
+const DisconnectUserFromDeviceService = new GQLCustomSchema('DisconnectUserFromDeviceService', {
+    types: [
+        {
+            access: true,
+            type: `enum PushTransportType { ${PUSH_TRANSPORT_TYPES.join(' ')} }`,
+        },
+        {
+            access: true,
+            type: 'input DisconnectUserFromDeviceInput { dv: Int!, sender: SenderFieldInput!, deviceId: String!, pushTransport: PushTransportType! }',
```

Comment:
you don't need a push token for disconnection

Reply:
It's pushTransport, not token
there could be deviceId collision between different transport systems, so deviceId should always come together with pushTransport